commands_buy.go

```diff
@@ -58,5 +59,49 @@ func (b buyCmd) ToAuditEntry() string {
 }
 
 func (b buyCmd) Execute() {
-	consoleLog.Warning("Not implemented: BUY")
+	consoleLog.Debug("Starting Buy")
+	if _, accountExists := accountStore[b.userID]; !accountExists {
+		consoleLog.Errorf("No account for %s exists", b.userID)
+		return
+	}
+
+	userAccount := accountStore[b.userID]
+
+	if userAccount.balance.ToFloat() < b.amount.ToFloat() {
```

Comment:
Perhaps, but I feel that the fact that one of the parameters is "amount" makes this clear.